library/std/src/os/linux/net.rs

```diff
@@ -0,0 +1,70 @@
+//! Linux-specific extensions to primitives in the [`std::net`] module.
+//!
+//! [`std::net`]: crate::net
+
+use crate::io;
+use crate::net;
+use crate::sealed::Sealed;
+use crate::sys_common::AsInner;
+
+/// Os-specific extensions for [`TcpStream`]
+///
+/// [`TcpStream`]: net::TcpStream
+#[unstable(feature = "tcp_quickack", issue = "96256")]
+pub trait TcpStreamExt: Sealed {
+    /// Sets the value of the `TCP_QUICKACK` option on this socket.
+    ///
+    /// If set, acks are sent immediately, rather than delayed if needed in
+    /// accordance to normal TCP operation. This flag is not permanent, it
+    /// only enables a switch to or from quickack mode.  Subsequent operation
+    /// of the TCP protocol will once again enter/leave quickack mode
+    /// depending on internal protocol processing and other factors.
+    /// (since Linux 2.4.4)
```

Comment:
This seems copied from the manpage. I don't know if that's ok from the licencing perspective.

Reply:
It isn't.